kafka.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+
+	"github.com/segmentio/kafka-go"
+	log "github.com/sirupsen/logrus"
+)
+
+// ProducerConfig configures the producer with needed kafka information
+type ProducerConfig struct {
+	Brokers []string
+	Topic   string
+	Async   bool
+}
+
+// Producer consumes the in channel and sends the message
```

Comment:
`Producer` does not sound like a function name. This function isn't returning anything; maybe we name it something like `ProduceMessages`?

```go
// ProduceMessages reads byte slices from the channel and writes them as messages to a Kafka topic.
```